apps/api/src/app/events/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -142,4 +140,51 @@ export class CreateNotificationJobs {
 
     return steps;
   }
+
+  private calculateExpireAt(command: CreateNotificationJobsCommand) {
+    const delayedSteps = command.template.steps.filter(
+      (step) => step.template?.type === StepTypeEnum.DIGEST || step.template?.type === StepTypeEnum.DELAY
+    );
+
+    const delay = delayedSteps
+      .map((step) => this.calculateDelayAmount(step, command.payload, command.overrides))
+      .reduce((partialSum, a) => partialSum + a, 0);
```

Comment:
Would really suggest to avoid naming things with single letters:
```suggestion
      .reduce((sum, delayAmount) => sum + delayAmount, 0);
```

Or similar, this becomes easier to read, wdyt?